app/components/primer/alpha/dialog.rb

```diff
@@ -125,8 +125,7 @@ def initialize(
         @position_narrow = position_narrow
         @visually_hide_title = visually_hide_title
 
-        @system_arguments[:tag] = "modal-dialog"
-        @system_arguments[:role] = "dialog"
+        @system_arguments[:tag] = "dialog"
```

Comment:
Why is a role no longer necessary?

Reply:
`<dialog>` has the implicit aria role of `dialog`. `modal-dialog` (being a custom element) had no implicit role.